documentation/modules/auxiliary/admin/dcerpc/cve_2020_1472_zerologon.md

```diff
@@ -43,20 +40,134 @@ this value. If this value is invalid the module will fail when making a Netlogon
 ### PASSWORD
 
 The hex value of the original machine account password. This value is typically recovered from the target system's
-registry (using a tool like secretsdump) after successfully setting the value to blank within Active Directory using
-this module and the default `REMOVE` action.
+registry (such as with the `auxiliary/gather/windows_secrets_dump` module) after successfully setting the value to blank
+within Active Directory using this module and the default `REMOVE` action.
 
 This value is only used when running the module with the `RESTORE` action.
 
 ## Scenarios
 
 ### Windows Server 2019
 
+First, set the machine account password to the empty value
```

Comment:
You're right, it can be a little confusing since the password is still technically present. Referring to it as an "empty string" is probably the most accurate and descriptive terminology. I had tried to stay consistent but was using empty & blank as synonyms for this. I'll update this and other references for clarity.